pkg/openstack/loadbalancer.go

```diff
@@ -727,25 +727,52 @@ func getSubnetIDForLB(network *gophercloud.ServiceClient, node corev1.Node, pref
 	return "", cpoerrors.ErrNotFound
 }
 
+// isPortMember returns true if IP and subnetID are one of the FixedIPs on the port
+func isPortMember(port PortWithPortSecurity, IP string, subnetID string) bool {
+	for _, fixedIP := range port.FixedIPs {
+		if (subnetID == "" || subnetID == fixedIP.SubnetID) && IP == fixedIP.IPAddress {
+			return true
+		}
+	}
+	return false
+}
+
 // applyNodeSecurityGroupIDForLB associates the security group with all the ports on the nodes.
```

Comment:
I think this comment should be modified, It not all the ports now.

Reply:
Nice catch, fixed!